src/stylesheets/components/_site-search.scss

```
@@ -136,9 +186,18 @@ $icon-size: 40px;
 .app-site-search__menu--overlay {
   position: absolute;
   z-index: 100;
-  top: 100%;
+  top: calc(100% - $govuk-border-width-form-element);
   left: 0;
-  box-shadow: rgba(govuk-colour("black"), 0.256863) 0 2px 6px; // stylelint-disable-line number-max-precision
+  // Creates a fake extra portion of the menu above the scrollable area to
+  // provide some space between the list items highlight-able top edge and the
+  // bottom of the input.
+  border-top: govuk-colour("white") solid govuk-spacing(1);
+  box-shadow: rgba(govuk-colour("black"), 0.256863) 0 govuk-spacing(1) govuk-spacing(1); // stylelint-disable-line number-max-precision
```

May want to interpolate function too.

```suggestion
  box-shadow: rgba(#{govuk-colour("black")}, 0.256863) 0 govuk-spacing(1) govuk-spacing(1); // stylelint-disable-line number-max-precision
```

---

So doing this one gives me an error:

>`sass.Exception [Error]: $color: #0b0c0c is not a color.`

I don't understand why. `#0b0c0c` _is_ a colour so 🤷🏻 I wonder if this is some weird string thing.